config/config.py

```diff
@@ -97,4 +91,4 @@ def get_google_sub():
     @staticmethod
     def get_max_message_size():
         if 'CAMPHOR_TREE_MAX_SIZE' in os.environ:
-            return os.environ['CAMPHOR_TREE_MAX_SIZE']
+            return int(os.environ['CAMPHOR_TREE_MAX_SIZE'])
```

Comment:
maybe you could do a .get() with a default, e.g. `return int(os.environ.get("CAMPHOR_TREE_MAX_SIZE","200"))` to ensure this never returns `None`?

Reply:
That is a great idea :D
I have gone through and implemented this advice throughout `config.py`